packages/validation/src/vpcs.schema.ts

```diff
@@ -127,16 +138,93 @@ export const updateVPCSchema = object({
   description: string(),
 });
 
-export const createSubnetSchema = object().shape(
+const VPCIPv6Schema = object({
+  range: string()
+    .optional()
+    .test({
+      name: 'IPv6 prefix length',
+      message: 'Must be the prefix length 52, 48, or 44 of the IP, e.g. /52',
+      test: (value) => {
+        if (value && value.length > 0) {
+          vpcsValidateIP({
+            value,
+            shouldHaveIPMask: true,
+            mustBeIPMask: false,
+          });
+        }
+      },
+    }),
+});
+
+const VPCIPv6SubnetSchema = object({
+  range: string()
+    .required()
+    .test({
+      name: 'IPv6 prefix length',
+      message: 'Must be the prefix length (52-62) the IP, e.g. /52',
+      test: (value) => {
+        if (value && value !== 'auto' && value.length > 0) {
+          vpcsValidateIP({
+            value,
+            shouldHaveIPMask: true,
+            mustBeIPMask: false,
+            isIPv6Subnet: true,
+          });
+        }
+      },
+    }),
+});
+
+// @TODO VPC: Delete this when IPv6 is in GA
```

Comment:
```suggestion
// @TODO VPC IPv6: Delete this when IPv6 is in GA
```

(just because there are several VPC projects underway)